.travis.yml

```diff
@@ -32,39 +33,49 @@ stages:
 jobs:
   include:
     - name: "Static checks (no pylint, no licence check)"
+      python: "3.5"
       stage: pre-test
-      script: ./scripts/ci/ci_run_all_static_tests_except_pylint_licence.sh
+      script: ./scripts/ci/ci_run_all_static_tests_except_pylint_checklicence.sh
     - name: "Check licence compliance for Apache"
+      python: "3.5"
       stage: pre-test
-      script: ./scripts/ci/ci_check_license.sh
+      script: ./scripts/ci/ci_run_checklicence.sh
     - name: "Pylint checks"
+      python: "3.5"
       stage: pre-test
       script: ./scripts/ci/ci_run_all_static_tests_pylint.sh
     - name: "Build documentation"
+      python: "3.5"
       stage: pre-test
       script: ./scripts/ci/ci_docs.sh
     - name: "Tests postgres kubernetes python 3.6 (persistent)"
-      env: BACKEND=postgres ENV=kubernetes KUBERNETES_VERSION=v1.15.0 KUBERNETES_MODE=persistent_mode
+      env: BACKEND=postgres KUBERNETES_MODE=persistent_mode RUN_KUBERNETES_TESTS=true
       python: "3.6"
       stage: test
+      script: travis_wait 30 "./scripts/ci/ci_run_airflow_testing.sh"
     - name: "Tests postgres kubernetes python 3.6 (git)"
-      env: BACKEND=postgres ENV=kubernetes KUBERNETES_VERSION=v1.15.0 KUBERNETES_MODE=git_mode
```

Comment:
Do we not support more than one version of Kube in tests anymore?

Reply:
For now no, but you are right we should bring this back. The KUBERNETES_VERSION we have  now in the Dockerfile turned into kubectl version (which is not the same as kubernetes version - it's just client version). 

Kind can actually install different versions of kubernetes - depending which "kindest" image you use. For now we are using the latest released version of Kubernetes (we do not specify image when we install kubernetes with kind).

I looked into the scripts and I will update them.  I will change the current KUBERNETES_VERSIONS to be properly named KUBECTL_VERSION instead and will provide a way to choose KUBERNETES_VERSION.

Reply:
I will add kubernetes version as extra argument of the kind setup. It's not part of the image (Kind installs kubernetes only after the image is started) so we will be able to add it as breeze script/environment variables when we run the script.